src/app/modal/components/lose-changes/lose-changes.component.scss

```diff
@@ -0,0 +1,29 @@
+@import "node_modules/govuk-frontend/govuk/all";
+
+.mat-dialog-content {
+  overflow: hidden;
+}
+
+.title {
+  color: #0B0C0C;
+  font-size: 36px;
+  font-weight: 700;
+  line-height: 43px;
+  width: 705px;
+  text-align: left;
+}
+
+.cancel {
+  color: #005EA5;
```

Comment:
Couldn't find this colour defined, i assume this is the greyish secondary colour?

Maybe we can declare it if it is not present
I assume the CTA will look like this?

![image](https://user-images.githubusercontent.com/62697030/86018548-6d5f4700-ba1d-11ea-9fed-d4fa312a2fad.png)



govuk-colour("light-grey") could be what we wanted perhaps

Reply:
This is actually looks like a link. I tried using the `govuk-link` class. It was behaving properly. I've updated the css to use `$govuk-link-colour`.